src/components/OSCALControlProse.js

```diff
@@ -55,17 +55,30 @@ function getParameterLabel(parameters, parameterId) {
 /**
  * Finds a parameter setting in a component statement
  * @param {Object} statementByComponent
+ * @param {Object} implReqSetParameters
  * @param {String} parameterId
  * @returns the parameter label
  */
-function getParameterValue(statementByComponent, parameterId) {
+function getParameterValue(
+  statementByComponent,
+  implReqSetParameters,
+  parameterId
+) {
   // Locate matching parameter to parameterId
-  if (!statementByComponent["set-parameters"]) {
+  let foundParameterSetting = null;
+  // If NIST, locate by by-component
```

Comment:
Here as well, not necessarily specific to NIST vs FedRAMP.

Reply:
Gotcha, will note this as a case when the `set-parameters` are found in the `by-component`